photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataSet.scala

```diff
@@ -65,16 +67,33 @@ protected[ml] class RandomEffectDataSet(
    * @return The dataset with updated offsets
    */
   override def addScoresToOffsets(scores: CoordinateDataScores): RandomEffectDataSet = {
-    val scoresGroupedByRandomEffectId = scores
+
+    val scoresGroupedByRandomEffectId: RDD[(REId, Array[(UniqueSampleId, Double)])] = scores
       .scores
       .join(uniqueIdToRandomEffectIds)
-      .map { case (uniqueId, (score, localId)) => (localId, (uniqueId, score)) }
+      .map { case (sampleId, (score, localREId)) => (localREId, (sampleId, score)) }
```

Comment:
Because of how widespread `uniqueId` is currently in our code and docs, it might be safer to rename to `uniqueSampleId` or to keep `uniqueId`